### PR TITLE
Add basic gtksourceview language-spec

### PR DESCRIPTION
**Pull request by rusty-snake**
Tested with org.gnome.TextEditor.

The gtksourceview language-spec hasn't changed between gtksourceview 3, 4 and 5 AFAIK so it should also work on older systems if you copy/link the file in the right places.

### CODE REVIEW

File: contrib/gtksourceview-5/language-specs/firejail-profile.lang [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    <context id="command-with-args">
+      <match>(?P&lt;command&gt;\%{commands-with-arguments}) (?P&lt;args&gt;.+)</match>
+      <include>
+        <context sub-pattern="command" style-ref="command"/>
+      </include>
+    </context>
+
+    <context id="command-without-args">
+      <match>(?P&lt;command&gt;\%{commands-without-arguments})</match>
```
Comment (rusty-snake) [inline contrib/gtksourceview-5/language-specs/firejail-profile.lang:49]:
```suggestion
      <match dupnames="true>(?P&lt;command&gt;\%{commands-without-arguments})</match>
```

File: contrib/gtksourceview-5/language-specs/firejail-profile.lang [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    <context id="command-with-args">
+      <match>(?P&lt;command&gt;\%{commands-with-arguments}) (?P&lt;args&gt;.+)</match>
+      <include>
+        <context sub-pattern="command" style-ref="command"/>
+      </include>
+    </context>
+
+    <context id="command-without-args">
+      <match dupnames="true>(?P&lt;command&gt;\%{commands-without-arguments})</match>
```
Comment (rusty-snake) [inline contrib/gtksourceview-5/language-specs/firejail-profile.lang:49]:
```suggestion
      <match dupnames="true">(?P&lt;command&gt;\%{commands-without-arguments})</match>
```

**Comment by rusty-snake** [comment]
OT: `Build CI / build_and_test` fails sometimes, re-run helps.

**Review by kmk3 — COMMENTED** [review]
Suggestion: Add `'contrib/gtksourceview-5/**'` to `paths-ignore` on every
workflow except for build.yml (similarly to the vim files).

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -39,6 +39,7 @@ If you add a new command, here's the checklist:
  - [ ] Update manpages: firejail(1) and firejail-profile(5)
  - [ ] Update shell completions
  - [ ] Update vim syntax files
+ - [ ] Update gtksourceview langauge spec
```
Comment (glitsj16) [inline CONTRIBUTING.md:42]: Typo: ... `language` spec

**Review by glitsj16 — APPROVED** [review]
Approving. Just one nitpick comment.

**Review by kmk3 — APPROVED** [review]
LGTM.

**Comment by kmk3** [comment]
By the way, I don't know if you're planning to add any more syntax files, but I
would ask to avoid adding many of them until we can generate them with the
commands in place, as more syntax files make adding/removing/reviewing commands
more cumbersome.

I have one branch that does something directly related to that; I'll see if I
can add a target to generate these files later.

**Pull request merged**